modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -78,7 +78,7 @@ def upload_form_526
           pending_claim = ClaimsApi::AutoEstablishedClaim.pending?(params[:id])
 
           if pending_claim && (pending_claim.form_data['autoCestPDFGenerationDisabled'] == true)
-            pending_claim.set_file_data!(documents.first, params[:doc_type])
+            pending_claim.set_file_data!(documents.first, EVSS_DOCUMENT_TYPE)
```

Comment:
I forgot to add this in the [previous PR](https://github.com/department-of-veterans-affairs/vets-api/pull/9671).  
So I'm handling it here now.